<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.2 (Python 3.12.5 on win32)" generated="2025-04-30T10:48:42.523806" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\HERA DWI PRADITA\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-04-30T10:48:42.651819" elapsed="4.914414"/>
</kw>
<status status="PASS" start="2025-04-30T10:48:42.649816" elapsed="4.916417"/>
</test>
<status status="PASS" start="2025-04-30T10:48:42.528799" elapsed="5.040437"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Robot Interactive Console" id="s1" pass="1" fail="0" skip="0">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-04-30T10:48:42.503815" level="WARN">Error in file 'c:\Users\HERA DWI PRADITA\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
